### PR TITLE
Support chrome path being passed as env var for windows

### LINKED ISSUE #19: It Is Unable to determine Google Chrome Install Location
https://prnt.sc/ulbs5b
Not very tech-sabby apologies if this is a simple issue

IANMARMOUR thank you for doing god's work!

### PR DESCRIPTION
**Pull request by NickOttrando**
Resolves #19, improves #5

### CODE REVIEW

**Comment by NickOttrando** [comment]
@ianmarmour did you want to support this functionality for all other OSes? Or just windows for now? I assume vast majority of users are on windows for this.

File: README.md [hunk 1]
```diff
@@ -81,6 +81,16 @@ set TWILIO_SOURCE_NUMBER=YOUR_TWILIO_SERVICE_NUMBER_HERE
...
 ```
 
+#### Chrome install path configuration
+
+If you encounter an error about your chrome path but it is installed, set the `CHROME_PATH` env var to the location of its exe.
+For example "C:/Program Files/Google/Chrome/Application/chrome.exe".
+You can find the path by right clicking on chrome in your start menu and choosing "Open File Location", right click the
+.exe file, select 'Properties', and copy and paste the Target.
+```
+set CHROME_PATH=PATH_TO_YOUR_CHROME.EXE
```
Comment (ianmarmour) [inline README.md]: Nitpick,

set CHROME_PATH=PATH/TO/YOUR/CHROME.EXE

File: internal/browser/browser.go [hunk 2]
```diff
@@ -38,6 +38,13 @@ func exists(name string) bool {
 
 //getWindowsChromeLocation Determines different Google Chrome install locations.
 func getWindowsChromeLocation() (string, error) {
+	// if user set chrome path env, check that first
+	chromePath, chromePathFound := os.LookupEnv("CHROME_PATH")
```
Comment (ianmarmour) [inline internal/browser/browser.go]: Lets change this environmental variable to be NVIDIA_CLERK_CHROME_PATH I would rather not have users possibly clutter other env uses.

**Review by ianmarmour — CHANGES_REQUESTED** [review]
Two minor changes, otherwise LGTM

**Review by ianmarmour — APPROVED** [review]
LGTM to me.

**Pull request merged**